.github/actions/common/run-testsuite/action.yaml

```diff
@@ -15,7 +15,7 @@ inputs:
     required: true
   prefix:
     description: 'prefix'
-    required: false
+    required: true
```

Comment:
This will prevent us from passing a blank prefix. Is that intended?

Reply:
I can't remember if that would prevent passing in `prefix == ''` even if we explicitly pass in `prefix = ${{ inputs.prefix }}`. I'll leave it as `required: false` for now until further testing

Reply:
Oh whoops - I thought this was a manual invocation.
I'm not sure either